Sources/Examples/RouteGuide/Model/route_guide.grpc.swift

```diff
@@ -0,0 +1,152 @@
+//
+// DO NOT EDIT.
+//
+// Generated by the protocol buffer compiler.
+// Source: route_guide.proto
+//
+
+//
+// Copyright 2018, gRPC Authors All rights reserved.
```

Comment:
(Unrelated to this PR) I guess we should update the copyright string in our protoc plugin soon

Reply:
Good point but should this even be our copyright header? SwiftProtobuf uses the copyright header from the source Proto file: https://github.com/apple/swift-protobuf/blob/da75a93ac017534e0028e83c0e4fc4610d2acf48/Sources/protoc-gen-swift/FileGenerator.swift#L71

I'll leave this unchanged for now and take a look into what other gRPC plugins do.

Reply:
SHTM